lib/taurus/qt/qtgui/taurusgui/paneldescriptionwizard.py

```diff
@@ -48,6 +47,11 @@
 __all__ = ["PanelDescriptionWizard"]
 
 
+def modelChooserDlg(*args, **kwargs):
```

Comment:
If possible, I would prefer to avoid adding public functions at the module level.

In this case I think that we can simply remove this method and add the import at the beginning of `showModelChooser()`